lib/graphql/execution/interpreter/handles_raw_value.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module GraphQL
+  module Execution
+    class Interpreter
+      # Wrapper for raw values
+      class RawValue
+        attr_reader :object
+
+        def initialize(obj = nil)
+          @object = obj
+        end
+
+        alias_method :resolve, :object
+      end
+
+      # Allows to return "raw" value from the resolver
+      module HandlesRawValue
+        def raw_value(obj = nil)
```

Comment:
What's the use case for `raw_value()` (without any argument)? How about requiring a value instead?
```suggestion
        def raw_value(obj)
```